drivers/starburst/src/metabase/driver/implementation/execute.clj

```diff
@@ -121,11 +121,12 @@
 
 (defmethod sql-jdbc.execute/read-column-thunk [:starburst Types/TIMESTAMP_WITH_TIMEZONE]
   [_ ^ResultSet rset _ ^long i]
-  ;; Converts TIMESTAMP_WITH_TIMEZONE to instant, then to OffsetDateTime with default time zone.
+  ;; Converts TIMESTAMP_WITH_TIMEZONE to java.time.ZonedDateTime with default time zone.
```

Comment:
is this comment true? Are are converting it to a `OffsetDateTime` in the end? Maybe we can update comment with updated details as to what's happening and why.

Reply:
Good catch. Fixed.